.github/workflows/validator.yml

```diff
@@ -0,0 +1,72 @@
+
+name: Dataset Validator
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  Validate:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+
+    #- name: Update software
+    #  run: |
+    # do we want to do this? it's helpful but also slow.
+    #    sudo apt-get update &&
+    #    sudo DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade &&
+    #    sudo DEBIAN_FRONTEND=noninteractive apt-get autoremove
+
+    - name: Install git-annex
+      run: |
+        sudo apt-get install -y git-annex
+        git config --global annex.thin true
+
+    - name: Install bids-validator
+      run: |
+        # install proper NodeJS for bids-validator
+        curl -sL https://deb.nodesource.com/setup_current.x | sudo -E bash -
+        sudo apt-get install -y nodejs
+        sudo npm install -g bids-validator
+
+    - name: Install spine-generic for acquisition parameters check
+      run: pip install spinegeneric@git+https://github.com/spine-generic/spine-generic.git@v2.3
```

Comment:
why not the latest version?

Reply:
Good question!  It's been like this for a while.

Funnily enough your name is on the commit that first added that: 93a49a50360ab633edcc7e58acc3171d8f4ee331

Let's change it to `master`?

Reply:
```suggestion
      run: pip install  #spinegeneric@git+https://github.com/spine-generic/spine-generic.git@master
```